modules/microshift-restart-ovnkube-master.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-restart-ovnkube-master_{context}"]
+= Restarting the ovnkube-master pod
+
+The following procedure restarts the ovnkube-master pod.
+
+.Prerequisites
+
+* The OpenShift CLI (`oc`) is installed.
+* Access to the cluster as a user with the `cluster-admin` role.
+* A cluster installed on infrastructure configured with the OVN-Kubernetes network plugin.
+* The KUBECONFIG environment variable is set.
+
+.Procedure
+
+Run the commands listed in each step that follows to restart ovnkube-master pod.
```

Comment:
Should this be 
```suggestion
Run the commands listed in each step that follows to restart `ovnkube-master` pod.
```
And all other references to it

Reply:
Also, I have a suggested rewrite for clarity if you would like. I got a little confused because each step says "run the following command"  you could honestly remove "listed in each step" and i think that would help with the word redundancy. So,
```suggestion
Run the following commands to restart the ovnkube-master pod.
```

or 

```suggestion
Follow the following steps to restart the ovnkube-master pod.
```

Reply:
I went with "Use the following steps."